core/src/test/java/org/frankframework/pipes/CompressPipeTest.java

```diff
@@ -135,10 +139,15 @@ public void testExceptionForward() throws Exception {
 		pipe.setResultIsContent(true);
 		pipe.addForward(new PipeForward(PipeForward.EXCEPTION_FORWARD_NAME, "dummy"));
 
-		configureAndStartPipe();
-		PipeRunResult prr = doPipe(pipe, DUMMY_STRING_SEMI_COLON, session);
+		// Verify that behaviour does not change when using the ExceptionHandlingPipeProcessor for exception forwarding instead of handling it in the CompressPipe
+		ExceptionHandlingPipeProcessor ehpp = new ExceptionHandlingPipeProcessor();
+		CorePipeProcessor cpp = new CorePipeProcessor();
```

Comment:
Dit is de binneste processor, daar zit de exceptionhandling-pipe-processor niet omheen toch?

Reply:
Ok, nu snap ik hem. doPipe in de PipeTestBase roept inderdaad de processor niet aan (al is die er wel).